### ISSUE TITLE
Control Panel Title Tag: Include actual page in title tag

### ISSUE THREAD

**Issue by auntialias** [SocialEngine/phpv4-issues#259]
The Title tags generated when SuperAdmin is in Control Panel all read "Control Panel."
When accessing the Back Arrow menu, or viewing multiple browser tabs, it's impossible to tell what specific page you'd like to see.

Screenshot of tabs, back menu: Now _which_ page exactly did you want to navigate back to?

![google chrome-se-controlpanel](https://cloud.githubusercontent.com/assets/1919971/9510958/fa52bf40-4c25-11e5-83ab-fb1e41cb0781.png)

Suggested Change: Please include actual page contents in Title Tag.

Control Panel > Blogs Plugin
Control Panel > Layout Editor

Alternatively, shorten Control Panel to CP so that more meaningful content is visible.

CP | Blogs Plugin
CP | Layout Editor

or add the changeable part of the title _before_ displaying the words Control Panel

Blogs Plugin | Control Panel
Layout Editor | Control Panel

**Comment by pulsatemedia** [comment]
Thanks for posting this antialias.

Please implement this, So much time is wasted on the admin side due to this for both myself and my staff that help with the SE sites.

**Comment by auntialias** [comment]
Thanks for your comment. Usability is important. Very important. (There's definitely a usability through-line to my bug-reports)

**Comment by Anshuse** [comment]
Hi @auntialias , We've added this issue in our internal bug tracker and will resolve this in near future release. For now, you may add below code at your end and after making these changes, page name will be displayed in the browser history like "Control Panel- Theme Editor", "Control Panel- Layout Editor" etc. :

In "custom.csv" file at path: ```/application/languages/en/custom.csv```, add below code at the end of file:

```
"pagetitle-user-index-admin-manage";"Manage Members"
"pagetitle-authorization-index-admin-level";"Member Levels"
"pagetitle-network-index-admin-manage";"Networks"
"pagetitle-announcement-index-admin-manage";"announcement"
"pagetitle-core-mail-admin-message";"Email All Members"
"pagetitle-core-index-admin-report";"Abuse Reports"
"pagetitle-core-locale-admin-settings";"Locale Settings"
"pagetitle-core-general-admin-settings";"General Settings"
"pagetitle-user-index-admin-signup";"Signup Process"
"pagetitle-user-twitter-admin-settings";"Twitter Integration"
"pagetitle-user-facebook-admin-settings";"Facebook Integration"
"pagetitle-user-janrain-admin-settings";"Janrain Integration"
"pagetitle-activity-index-admin-settings";"Activity Feed Settings"
"pagetitle-core-spam-admin-settings";"Spam & Banning Tools"
"pagetitle-user-friends-admin-settings";"Friendship Settings"
"pagetitle-core-templates-admin-mail";"Mail Templates"
"pagetitle-core-settings-admin-mail";"Mail Settings"
"pagetitle-core-performance-admin-settings";"Performance & Caching"
"pagetitle-core-password-admin-settings";"Admin Password"
"pagetitle-core-index-admin-tasks";"Task Scheduler"
"pagetitle-storage-index-admin-services";"Storage System"
"pagetitle-activity-notifications-admin-settings";"Default Email Notofications"
"pagetitle-poll-index-admin-manage";"Poll"
"pagetitle-music-index-admin-manage";"Music"
"pagetitle-group-index-admin-manage";"Group"
"pagetitle-video-index-admin-manage";"Video"
"pagetitle-forum-index-admin-manage";"Forum"
"pagetitle-event-index-admin-manage";"Event"
"pagetitle-chat-index-admin-manage";"Chat"
"pagetitle-blog-index-admin-manage";"Blog"
"pagetitle-album-index-admin-manage";"Photo Album"
"pagetitle-classified-index-admin-manage";"Classified"
"pagetitle-core-index-admin-content";"Layout Editor"
"pagetitle-core-index-admin-themes";"Theme Editor"
"pagetitle-core-index-admin-files";"File & Media Manager"
"pagetitle-core-index-admin-language";Language Manager
"pagetitle-core-index-admin-menus";"Menu Editor"
"pagetitle-core-index-admin-ads";"Manage Ads"
"pagetitle-core-create-admin-ads";"Create Ads"
"pagetitle-core-affiliate-admin-settings";"Affiliate Program"
"pagetitle-core-viglink-admin-settings";"Viglink"
"pagetitle-payment-index-admin-index";"Transactions"
"pagetitle-payment-index-admin-settings";"Payment Settings"
"pagetitle-payment-index-admin-gateway";"Payment Gateways"
"pagetitle-payment-index-admin-package";"Plans"
"pagetitle-payment-index-admin-subscription"; Subscription
"pagetitle-core-index-admin-stats";"Statistics"
"pagetitle-core-referrers-admin-s";"Error Logs
"pagetitle-core-php-admin-system";"Server Information"
```

Thanks!

**Comment by auntialias** [comment]
Took me a while to implement this. But I did so yesterday. 

Immediate results: no change. 

Later on (next day), I checked again, and it worked as expected. Thanks. I guess the previous titles were cached or something?

**Comment by cgcb** [comment]
I fixed this specific issue by modding /application/modules/Core/layouts/scripts/admin.tpl, lines 37-39 from:

$this
      ->headTitle($this->translate("Control Panel"), Zend_View_Helper_Placeholder_Container_Abstract::PREPEND)
      ;

to

$this->headTitle($this->translate("Control Panel") . '-' . $this->translate($this->layout()->siteinfo['title']));

And in custom.csv, I added:

"Control Panel";"CP"

**Comment by DonnaScriptTechs** [comment]
This is fixed for 4.9.3 release when it is released.
